### PR TITLE
[CIS-798] Expose quoted message on `ChatMessage`

### PR DESCRIPTION
**Pull request by evsaev**
**This PR**:
- exposes `quotedMessage` on `ChatMessage`
- makes `quotedMessageId` computed from `quotedMessage`
- deprecates `quotedMessageId`

### CODE REVIEW

File: Sources/StreamChat/Deprecations.swift [hunk 1]
```diff
@@ -23,6 +23,14 @@ public extension _ChatChannel {
     var watchers: Set<_ChatUser<ExtraData.User>> { Set(lastActiveWatchers) }
 }
 
+public extension _ChatMessage {
+    /// Quoted message id.
+    ///
+    /// If message is inline reply this property will contain id of the message quoted by this reply.
+    @available(*, deprecated, renamed: "quotedMessage.id")
+    var quotedMessageId: MessageId? { quotedMessage?.id }
```
Comment (VojtaStavik) [inline Sources/StreamChat/Deprecations.swift:31]: Did you test if this works properly? Shouldn't the suggestion be `quotedMessage?.id ` such that it compilers right away?

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - It's safe now to use `ChatChannel` and `ChatMessage` across multiple threads [#984](https://github.com/GetStream/stream-chat-swift/pull/984)
 - Web socket reconnection logic better handles the "no internet" errors [#970](https://github.com/GetStream/stream-chat-swift/pull/970)
 
+### ✅ Added
+- Expose the entire quoted message on `ChatMessage` instead of it's `id` [#992](https://github.com/GetStream/stream-chat-swift/pull/992)
```
Comment (VojtaStavik) [inline CHANGELOG.md:13]: nit: `it's` -> `its`

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,6 +9,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
...
+
+### ⛔️ Deprecated
+- `ChatMessage`'s property `quotedMessageId` is now deprecated. Use `quotedMessage.id` instead [#992](https://github.com/GetStream/stream-chat-swift/pull/992)
```
Comment (VojtaStavik) [inline CHANGELOG.md:16]: nit: You can just write `ChatMessage.quotedMessageId` is .... it's easier to read :) Also, `quotedMessage.id` doesn't compile, it must be `quotedMessage?.id`

**Review by VojtaStavik — APPROVED** [review]
LGTM 👍 Just some nits a documentation tweaks

File: Sources/StreamChat/Deprecations.swift (hunk 1 above)
Comment (evsaev) [reply Sources/StreamChat/Deprecations.swift]: The `renamed: "quotedMessage?.id"` doesn't compile 🤷‍♂️ Replaced it with `message: "Use quotedMessage?.id instead"`

File: CHANGELOG.md (hunk 2 above)
Comment (evsaev) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 3 above)
Comment (evsaev) [reply CHANGELOG.md]: done ✅

**Pull request merged**